src/main/java/api/apiwrappers/PetsApiWrapper.java

```diff
@@ -0,0 +1,61 @@
+package api.apiwrappers;
+
+import java.io.IOException;
+import api.CommonApiWrapper;
+import api.domain.pets.*;
+
+public class PetsApiWrapper extends CommonApiWrapper{
+    public Pet postPet(Pet pet) throws IOException {
+        String endpoint = baseUrl + "/v2/pet";
+
+        return (Pet)post(pet, endpoint);
+    }
+
+    public Pet postPetById(Pet pet) throws IOException {
+        String endpoint = baseUrl + "/v2/pet/" + pet.getId();
+
+        return (Pet)post(pet, endpoint);
+    }
+
+    public Pet postPetUploadImage(Pet pet) throws IOException {
+        String endpoint = baseUrl + "/v2/pet/" + pet.getId() + "/uploadImage";
+
+        return (Pet)post(pet, endpoint);
+    }
+
+    public Pet putPet(Pet pet) throws IOException {
+        String endpoint = baseUrl + "/v2/pet";
+
+        return (Pet)put(pet, endpoint);
+    }
+
+    public Pet getPetById(Pet pet) throws IOException {
+        String endpoint = baseUrl + "/v2/pet/" + pet.getId();
+
+        return (Pet)get(pet, endpoint);
+    }
+
+    public Pet getPetFindByStatus(Pet pet) throws IOException {
+        String endpoint = baseUrl + "/v2/pet/findByStatus";
+
+        return (Pet)get(pet, endpoint);
+    }
+
+    public void deletePetById(Pet pet) throws IOException {
+        String endpoint = baseUrl + "/v2/pet/" + pet.getId();
+
+        delete(endpoint);
+    }
+
+    public int getGetPetResponse(){
```

Comment:
Method name should end in ...ResponseStatus(), to reflect that it returns a status code.

Reply:
Updated name methods:  `getGetPetResponseStatus()`, `getPostPetResponseStatus()` and `getDeletePetResponseStatus()`